### PR TITLE
(#3003) update axios package and cts packages using axios

### LINKED ISSUE #3003: Enabler: Axios version update to fix high severity security warning
GitHub's dependabot has alerted us to a high severity vulnerability in Axios versions prior to 0.21.1.  We are currently using version 0.19 in docroot/profiles/custom/cgov_site/themes/custom/cgov/package-lock.json

The Axios version should be updated to at least v0.21.1 and the app retested.

[The vulnerability is CVE-2020-28168](https://github.com/advisories/GHSA-4w2v-q235-vp99)

Secondarily, use the results of `npm audit` to determine what other packages might need to be updated.

### PR DESCRIPTION
**Pull request by arcepaul**
Closes #3003 .

- Updated axios to 0.21.1
- Updated CTS app and search client (which use axios) to latest patched packages

ODE for testing(when ready)
https://ncigovcdode404.prod.acquia-sites.com/
https://ncigovcdode404.prod.acquia-sites.com/about-cancer/treatment/clinical-trials/search

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/package-lock.json [hunk 1]
```diff
@@ -2545,6 +2503,14 @@
...
           }
         },
+        "axios": {
+          "version": "0.19.2",
+          "resolved": "https://registry.npmjs.org/axios/-/axios-0.19.2.tgz",
+          "integrity": "sha512-fjgm5MvRHLhx+osE2xoekY70AhARk3a6hkN+3Io1jc00jtquGvxYlKlsFUhmUET0V5te6CcZI7lcv2Ym61mjHA==",
+          "requires": {
+            "follow-redirects": "1.5.10"
+          }
+        },
```
Comment (blairlearn) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/package-lock.json:2513]: This surprises me.  Shouldn't this be axios 0.21.1 like the others?

This is from a list of required packages for the glossary app.  That also surprises me; when we switched to the generic app modules, I thought we would no longer need to reference those packages in the platform's build.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/glossary/GlossaryApp.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
  */
 
 import initializeGlossaryApp from 'Libraries/glossary/glossary';
-import './GlossaryApp.scss';
+// import './GlossaryApp.scss';
```
Comment (blairlearn) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/glossary/GlossaryApp.js:7]: Is this meant to be removed?  Or is this a "Trying something and forgot to uncomment it?"

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/sitewideSearch/SitewideSearchApp.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
  */
 
 import initializeSitewideSearchApp from "Libraries/sitewideSearch/sitewideSearchApp";
-import "./SitewideSearchApp.scss";
+// import "./SitewideSearchApp.scss";
```
Comment (blairlearn) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/sitewideSearch/SitewideSearchApp.js:7]: Is this meant to be removed? Or is this a "Trying something and forgot to uncomment it?"

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/sitewideSearch/sitewideSearchApp.js [hunk 4]
```diff
@@ -3,37 +3,58 @@
...
+const metaTags = getMetaData(
+  [
+    ['name', 'dcterms.subject'],
+    ['name', 'dcterms.isPartOf'],
+    ['name', 'dcterms.issued'],
+    ['property', 'og:site_name'],
+    ['property', 'og:title']
+  ],
+  document
+);
```
Comment (blairlearn) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/sitewideSearch/sitewideSearchApp.js:30]: I'm not sure what I'm seeing here.  This file should be going away once the Drupal module is completed, and yet, there are changes being made.

That shouldn't affect anything if the code never executes (and the changes to initialize method should prevent that), but I'm confused.  Some of this is just reformatting, but there's new code as well (e.g. the various Utilities imports).

**Review by blairlearn — CHANGES_REQUESTED** [review]

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/glossary/GlossaryApp.js (hunk 2 above)
Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/glossary/GlossaryApp.js:7]: yes, if left in sass compilation stops dead in its tracks and the build will fail.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/sitewideSearch/SitewideSearchApp.js (hunk 3 above)
Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/sitewideSearch/SitewideSearchApp.js:7]: same as previous

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/sitewideSearch/sitewideSearchApp.js (hunk 4 above)
Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/sitewideSearch/sitewideSearchApp.js:30]: No it is all just code reformatting here.  The utilities imports was broken out on multiple lines.

Comment (blairlearn) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/sitewideSearch/sitewideSearchApp.js:30]: Oh, I see. Dang that's confusing! 😕

**Review by blairlearn — APPROVED** [review]

**Pull request merged**